### PR TITLE
Manage library and data set

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: sql_ddl/db.sql [hunk 1]
```diff
@@ -8,126 +8,114 @@ create type test_scenario_status as enum ('ACTIVE' , 'INACTIVE');
...
+    id           serial primary key  not null,
+    email        varchar(100) UNIQUE not null,
+    password     varchar(300)        not null,
+    name         varchar(100)        not null,
+    role         user_role           not null,
+    status       user_status         not null,
+    last_request timestamp
 );
 
 create table reset_token
```
Comment (KotyaraSingleCat) [inline sql_ddl/db.sql]: We delete this table

Comment (ghost) [reply sql_ddl/db.sql]: it is not my table and ddl was created by it`s owner, so I dont think I should delete it

**Review by danilos1 — APPROVED** [review]

**Review by HellaxA — COMMENTED** [review]
Code smells?

**Review by GorovDimon11 — APPROVED** [review]
Great

**Review by HellaxA — APPROVED** [review]

**Pull request merged**